### PR TITLE
makes audit field timestamps match

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/16047

this pr sets `now()` as the default value for all instances of `created_at` and `updated_at` fields in `moped_proj_` tables and refactors the audit field update trigger to only update the `updated_at` field on `UPDATE` (and not on `INSERT`)

## Testing
**URL to test:** 

local because it consists of database changes

**Steps to test:**

1. add a new phase to a project in moped
2. check the record in the `moped_proj_phases` table to confirm the `updated_at` timestamp matches the `created_at` timestamp by checking the last four decimal places
3. go to that record's parent project and confirm the `updated_at` timestamp matches its child record's timestamp fields
4. edit the phase you just added in moped by updating the description
5. confirm that both the record and its parent record's `updated_at` timestamps updated accordingly
6. repeat these steps for `moped_proj_milestones`, `moped_proj_funding`, `moped_proj_work_activity` (on the funding tab), `moped_proj_notes`, and `moped_proj_components`

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/metadata/tables.yaml [hunk 1]
```diff
@@ -2650,7 +2650,7 @@
           - milestone_order
           - project_id
           - is_deleted
-        comment: No insert permissions on audit fields
+      comment: No insert permissions on audit fields
```
Comment (tillyw) [inline moped-database/metadata/tables.yaml:2653]: all the edits in this file are styling fixes

**Comment by chiaberry** [comment]
when I add a new phase to a project, the moped_project `updated_at` and `updated_by_user_id` don't update. Is there a trigger to update the moped_project upon insert?

**Review by frankhereford — APPROVED** [review]
edit: Tilly - I think, based on some conversations in slack, the concern about bubbling up edit times may be a regression here. I will re-approve as soon as we settle on what should be the expected behavior. Thanks!

------

Tilly, this is great. Thank you for working though all this, and for putting together really helpful instructions. It looks clean, and it worked just like it should. 

Chia has a good question that she's posed. The update times bubbling up from the table being edited up to projects' edited times -- that's out of scope of this work? I observed the same thing. I'm going to approve this under the assumption that that is out of scope, but if not, please let me know, and I'll go back and figure out which of these have had the work to bubble up that edit time, and revise this review.

Here are queries I have accumulated during this testing, if they are helpful to others:

```
select project_id, phase_id, created_at, updated_at 
from moped_proj_phases 
order by created_at desc, project_phase_id desc;

select project_id, date_added, updated_at
from moped_project
order by updated_at desc;

select project_id, milestone_id , created_at, updated_at 
from moped_proj_milestones 
order by created_at desc, project_milestone_id desc;

select project_id, id, created_at, updated_at 
from moped_proj_work_activity  
order by created_at desc, id asc;

select project_id, project_note_id, created_at, updated_at 
from moped_proj_notes  
order by created_at desc, project_note_id desc;

select project_id, component_id, created_at, updated_at 
from moped_proj_components  
order by created_at desc, component_id desc;

select id, component_id, created_at, updated_at 
from feature_drawn_lines
order by created_at desc, id desc;

select id, component_id, created_at, updated_at 
from feature_drawn_points
order by created_at desc, id desc;

select id, created_at, updated_at, is_deleted 
from feature_intersections 
order by created_at desc;

select id, created_at, updated_at, is_deleted 
from feature_signals 
order by created_at desc;

select id, created_at, updated_at, is_deleted 
from feature_street_segments 
order by created_at desc;
```

**Comment by tillyw** [comment]
thanks y'all for your input and patience on this one. i've reworked this pr, narrowed the scope to the tables that use the trigger we've been discussing, and retested it. re-requesting review, thank you 🙏

**Review by chiaberry — APPROVED** [review]
thank you for sticking with it through all the back and forth

I see the timestamps how i expect them, the moped_project updates and the user ids all checking out.

**Review by mddilley — CHANGES_REQUESTED** [review]
This works spendidly! ✨ I tested locally and made updates and inserts on all the affected tables, and I'm seeing matching timestamps in the expected columns.

One change needed - I am getting an error when running the down migration. Could you check that out? Other than that, this is ready to 🚢 🙌

**Review by mddilley — APPROVED** [review]
Awesome!!! thanks Tilly! 🚢 🚀

**Pull request merged**